recipes_source/recipes_index.rst

```diff
@@ -407,7 +407,7 @@ Recipes are bite-sized, actionable examples of how to use specific PyTorch featu
 .. toctree::
    :hidden:
 
-   /recipes/recipes/loading_data_recipe
+   /recipes/loading_data_recipe
```

Comment:
since we are deleting it, just remove line 410

Reply:
I thought the ask was to deprecating and **creating a redirect from this tutorial to the data_tutorial.html**. Wouldn't this remove the tutorial entirely without a redirect?

Reply:
same as above.